docs/GettingStartedGuide.md

```diff
@@ -50,7 +50,7 @@ HERE OLP SDK consists of specific libraries, each with distinct functionality.
 
 [Volatile Layer Read example](./examples/nodejs-read-volatile-layer.md) shows how to retrieve layer metadata, partition metadata, and partition data from volatile layer using OLP SDK for TypeScript.
```

Comment:
Articles are missing.
Change to:
"[Volatile Layer Read example](./examples/nodejs-read-volatile-layer.md) shows how to retrieve layer metadata, partition metadata, and partition data from a volatile layer using the OLP SDK for TypeScript."